packages/frontend/src/utils/bigNumberConfig.ts

```diff
@@ -3,7 +3,7 @@ import { ApiQueryResults } from 'common';
 const bigNumberConfig = (data: ApiQueryResults | undefined) => {
     if (!data || !data.rows.length) return null;
     const metric: string = data.metricQuery.metrics[0];
-    const bigNumber: number | string = data.rows[0][metric];
+    const bigNumber: number | string = data.rows[0][metric].value.raw;
```

Comment:
We should use the formatted value for the big number visualisation, it'd look great!